spark-batch-prediction/spark-base/build-spark-image.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-spark_version=3.0.2
+spark_version=3.2.0
```

Comment:
did you test with this minor version upgrade?

Reply:
yes, tested and works